Appraisals

```diff
@@ -1,10 +1,25 @@
 appraise "rails41" do
-  gem 'rails', '~> 4.1.4'
-  gem 'compass-rails', '1.1.7'
+  gem 'rails',         '~> 4.1.4'
```

Comment:
dit blok kan er ook uit als we support droppen.

mogelijk nog wel aardig om een old_gems appraise toe te voegen met de nieuwste rails, maar de oudste mayor versies van alle gems die we zeggen te supporten.